packages/wrangler/src/index.tsx

```diff
@@ -577,6 +574,38 @@ function createCLIParser(argv: string[]) {
         !packageJsonContent.scripts?.publish &&
         shouldCreatePackageJson;
 
+      async function installPackages(
+        shouldRunInstall: boolean,
+        depsToInstall: string[]
+      ) {
+        if (hasRunPackageInstall) {
+          return;
+        }
+        //lets install the devDeps they asked for
+        //and run their package manager's install command if needed
+        if (depsToInstall.length > 0) {
+          const formatter = new Intl.ListFormat("en", {
+            style: "long",
+            type: "conjunction",
+          });
+          await packageManager.addDevDeps(...depsToInstall);
+          const versionlessPackages = depsToInstall.map((dep) =>
```

Comment:
I'm probably just complicating things here, but I wanted the log to be clean while still installing the correct wrangler version